File: java/core-fw-spring/src/main/java/com/firelord/spring/component/rpc/http/vo/RespErrItemVo.java
package com.firelord.spring.component.rpc.http.vo;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

/**
 * RespErrItemVo
 */
@ToString
public class RespErrItemVo {
    //#region Fields

    /**
     * error code
     */
    @Setter
    @Getter
    private String code;

    /**
     * error description
     */
    @Setter
    @Getter
    private String desc;

    /**
     * error reason
     */
    @Setter
    @Getter
    private String reason;

    /**
     * error advice
     */
    @Setter
    @Getter
    private String advice;

    //#endregion
}

File: cs/core-fw-spring/component/rpc/http/vo/RespErrItemVo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace core_fw_spring.component.rpc.http.vo
{
    public class RespErrItemVo
    {
        #region Fields

        private String code;
        private String desc;
        private String reason;
        private String advice;

        public string Code { get => code; set => code = value; }
        public string Desc { get => desc; set => desc = value; }
        public string Reason { get => reason; set => reason = value; }
        public string Advice { get => advice; set => advice = value; }

        #endregion
    }
}
